@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -1111,8 +1107,6 @@ export class TextElementsRenderer {
      * @param dataSourceTileList - List of tiles to be rendered for each data source.
      */
     private updateTextElements(dataSourceTileList: DataSourceTileList[]) {
-        logger.debug("updateTextElements");
-
```

Comment:
This log is useful to know when the text elements are updated due to cache or tile invalidation.

Reply:
yeah, I get that but if all the code starts to add these type of log messages then there's no way to enable the tracing and get something decent out of it. Can you add an additional check to enable these messages when you need them?